src/packages/core/src/base-resolver.ts

```diff
@@ -306,6 +307,13 @@ export function createBaseResolver<G extends WithId, D extends BaseDataEntity>(
 
 	@Resolver()
 	abstract class BaseResolver implements BaseResolverInterface {
+		static metadata: {
```

Comment:
this can be removed now?

Reply:
Yes, because the connection is in the constructor of the MikroBackendProvider class now